core/shared/src/main/scala/zio/stm/STM.scala

```diff
@@ -279,6 +279,22 @@ object STM {
   val unit: STM[Nothing, Unit] =
     ZSTM.unit
 
+  /**
+   * @see See [[zio.stm.ZSTM.validate]]
+   */
+  def validate[E, A, B](
+    in: Iterable[A]
+  )(f: A => STM[E, B])(implicit ev: CanFail[E]): STM[::[E], List[B]] =
+    ZSTM.validate(in)(f)
+
+  /**
+   * @see See [[zio.stm.ZSTM.validateFirst]]
+   */
+  def validateFirst[E, A, B](
+    in: Iterable[A]
+  )(f: A => STM[E, B])(implicit ev: CanFail[E]): STM[List[E], B] =
+    ZSTM.foreach(in)(f(_).flip).flip
```

Comment:
Don't we want this to just be an alias for the method on `ZSTM`?

Reply:
we do. you can slap me for that one